### DISCUSSION TITLE
invalid copilot token: missing token: 403

### DISCUSSION THREAD

**Discussion by MrOplus** [community/community#51753] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

from today's morning I'm receiving the following error : 
```
[INFO] [auth] [2023-04-02T14:28:59.017Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2023-04-02T14:28:59.018Z] GitHub Copilot could not connect to server. Extension activation failed: "At this time, Copilot is not available in your location"
```
The problem persists with/without VPN in Muscat, Oman

**Comment by lucas-low** [comment]
similar issue since last thursday. tried relogging across multiple devices not working and seems to find yours as the 1st issue so far on this problem. so its not isolated.

**Comment by Akash1134** [comment, marked as answer]
Hi @kooroshh and @lucas-low ,

Thanks for getting in touch.

Due to current trade controls, your GitHub account has been restricted from making use of Copilot. This is a region-based restriction and applies to all users located in some trade-restricted countries, being flagged for the same earlier, and is separate from other GitHub services. You can continue to use other GitHub services including your Pro plan. You can read more about it here:

[GitHub and Trade Controls](https://docs.github.com/en/site-policy/other-site-policies/github-and-trade-controls)

Sorry if that wasn't what you were hoping to hear.

If you feel this should not be the case, feel free to contact our Support team and [open a ticket ](https://support.github.com/request?tags=dotcom-footer)with them. Thanks!

**Comment by 2yxh** [comment]
I am having this issue in the UK...

**Comment by plswork1** [comment]
Hello!
Here in 2024 I have the same problem, I used the copilot two days ago and in this morning but when i am trying to use it again it gives me this error. How it can be solved?
Btw i am from Romania if thats the problem

**Reply by plswork1** [reply]
actually I did not pay attention of that instead of being written "At this time, Copilot is not available in your location" it is written "Contact Support. You are currently logged in as plswork1."
